sw/device/tests/rv_dm_access_after_wakeup.c

```diff
@@ -31,8 +31,13 @@ enum {
   kSoftwareBarrierTimeoutUsec = 1000000,
 };
 
-// This location will be update from SV
-static volatile const uint8_t kSoftwareBarrier = 0;
+// This variable will be updated by the DV environment.
+OT_SECTION(".rodata")
+static volatile const uint8_t kSoftwareBarrierDv = 0;
+
+// This variable will be updated by the test host.
+OT_SECTION(".data")
+static volatile const uint8_t kSoftwareBarrierHost = 0;
```

Comment:
I followed the example of `sw/device/tests/sysrst_ctrl_outputs_test.c`, and since we don't want C code to write this variable I think the `const` is useful.